packages/mobile/src/in-house-liquidity/index.ts

```diff
@@ -4,9 +4,13 @@ import KeyEncoder from 'key-encoder'
 import { compressedPubKey } from '@celo/utils/lib/dataEncryptionKey'
 import { hexToBuffer, trimLeading0x } from '@celo/utils/lib/address'
 
-interface DeleteFinclusiveBankAccountParams {
-  accountMTWAddress: string
-  dekPrivate: string
+const keyEncoder = new KeyEncoder('secp256k1')
+interface RequiredParams {
+  accountMTWAddress: string | null
+  dekPrivate: string | null
+}
```

Comment:
I don't think required params should be allowed to be null-- seems like the type and the name are at odds